landing_page/db/schema.rb

```diff
@@ -78,5 +88,6 @@
   end
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
+  add_foreign_key "archivedposts", "users"
```

Comment:
Why do you need this table?

Reply:
Resolved in commit 084d812ae08e7a35c71b40da5aca618a372678fa